test/integration/acceptance/get_roles_test.cpp

```diff
@@ -59,28 +52,34 @@ TEST_F(GetRoles, CanGetRoles) {
  * @then there is should be an exception
  */
 TEST_F(GetRoles, CanNotGetRoles) {
-    auto checkQuery = [](auto &queryResponse) {
-        ASSERT_ANY_THROW(boost::apply_visitor(
-                framework::SpecifiedVisitor<shared_model::interface::RolesResponse>(), queryResponse.get()));
-    };
-
-    auto query = TestUnsignedQueryBuilder()
-            .createdTime(iroha::time::now())
-            .creatorAccountId(kUserId)
-            .queryCounter(1)
-            .getRoles()
-            .build()
-            .signAndAddSignature(kUserKeypair)
-            .finish();
+  auto checkQuery = [](auto &queryResponse) {
```

Comment:
Rewrite "then" section. The same comment as for previous test case. In this test case, the description of "then" is outdated.